public/script.js

```diff
@@ -61,14 +68,24 @@ document.addEventListener("DOMContentLoaded", () => {
         .join(" ");
       liElement.remove();
 
-      // constで宣言しても配列操作はできるのでconstにしておく
-      const tmp = [];
-      for (i = 0; i < created_sentence.length; i++) {
-        if (created_sentence[i] !== textContent) {
-          tmp.push(created_sentence[i]);
-        }
-      }
-      created_sentence = tmp;
+      created_sentence = created_sentence.filter((e) => {
+        return e.message !== textContent;
+      });
     }
   });
 });
+
+async function _messageSave() {
+  const cocktails = [];
+  created_sentence.forEach((e) => {
+    cocktails.push({ name: String(e.name) });
+  });
+  const res = await fetch("/api/message", {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({ "cocktails": cocktails }),
+  });
+
+  const body = await res.text();
+  document.getElementById("message_save").href = "/check?id=" + body;
```

Comment:
```suggestion
  window.location = "/check?id=" + body;
```
既存の実装だと、もう一回クリックされないと遷移しないかも。
ボタンクリックした瞬間にページ遷移した方が体験が良さそう。